### PR TITLE
[Security][Validator] Add missing translations for Gallician (gl)

### LINKED ISSUE #41044: Missing translations for Galician (gl)
Hello,

There are some missing translation for Galician (gl). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Security/Core/Resources/translations/security.gl.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.gl.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.gl.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.gl.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by afaricamp**
| Q             | A
| ------------- | ---
| Branch?      | 4.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #41044 
| License       | MIT
| Doc PR        | symfony/symfony-docs#...
Added missing Gallician translation units for security and validators.

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.gl.xlf [hunk 1]
```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>Ligazón de inicio de sesión non válida ou caducada.</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
+                <target>Demasiados intentos de inicio de sesión errados, por favor, ténteo de novo en %minutes% minutos.</target>
```
Comment (esmiz) [inline src/Symfony/Component/Security/Core/Resources/translations/security.gl.xlf]: I think the word «minutos» should be singular here: «minuto»

**Comment by esmiz** [comment]
Thanks @afaricamp for this pull request! I think there is a minor issue to solve. One of the sentences should be singular, the word «minutos» should be «minuto». 

Status: Needs work

File: src/Symfony/Component/Security/Core/Resources/translations/security.gl.xlf (hunk 1 above)
Comment (afaricamp) [reply src/Symfony/Component/Security/Core/Resources/translations/security.gl.xlf]: Right! Fixed!

**Comment by fabpot** [comment]
Thank you @afaricamp.

**Pull request merged**